index.html

```diff
@@ -0,0 +1,383 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>YouTube Clone</title>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;400;500;600&display=swap"
+        rel="stylesheet">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link href="styles.css" rel="stylesheet">
+</head>
+
+<body>
+
+    <!--IN THIS TASK, I GROUP THE ENTIRE PAGE INTO 4 GROPUS
+    THE SIDEBAR...IT HAS ITS OWN SUBGROUP, THE HEADER...LEFT SIDE(MENU AND LOGO), THE SEARCH
+    BAR AND THE RIGHT BAR...ICONS..., THE CATEGORIES GROUP, AND BODY IS GROUPED INTO THREE-RECOMMENDED VIDEOS, 
+    SPATIAL SECTION AND OTHES 
+    I will use some custome inages and unsplash images 
+    I used google icons for the header and sidebar icons -->
+    <div class="header">
```

Comment:
It is recommended to use HTML semantic tags, which allow your code more understandable by better defining the different sections and layout of web pages. As a result, please use HTML semantic tags.